### ISSUE TITLE
Problem with use npm dependencies in custom hook

### ISSUE THREAD

**Issue by FherEnrique** [directus/directus#16874]
### Describe the Bug

When you used a dependency on npm.

for example: uuid when using it in the code shows strange errors
"{
   "message": "crypto.getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported",
   "extensions": {
     "code": "INTERNAL_SERVER_ERROR"
   }
}"

In the link that is shown, it gives a solution that I already implemented and it still does not work.
And it doesn't only happen with that library, it basically happens with all npm libraries.
(I am not a native speaker sorry for errors)

### To Reproduce

go into the custom-hook folder run `npm i xxxx` and then try to use it

### What version of Directus are you using?

9.16.1

### What version of Node.js are you using?

16.18.0

### What database are you using?

PG  12

### What browser are you using?

Brave

### How are you deploying Directus?

running locally

**Comment by rijkvanzanten** [comment]
Can you double check and confirm the node version you're using? UUIDjs requires a newer version (15+) as it relies on `crypto.getRandomValues()`

**Comment by azrikahar** [comment]
I would agree with @rijkvanzanten here, as it does feel like a Node version that may not be 15+.

However if you are still having this issue even after confirming it's Node 16+ as described in your issue detail, it would also be helpful if you can try this on the latest version, which would be `9.22.1` as of the writing of this response. 

Feel free to share the code for your hook extension if possible as well, and the version of `@directus/extensions-sdk` in your hook extension project.
